src/main/java/Duke.java

```diff
@@ -1,10 +1,236 @@
+import java.util.Scanner;
+
 public class Duke {
+    /** Array of assignments that is limited to 100 tasks */
+    private static final Task[] assignments = new Task[100];
+    /** Use to track the number of Task that is added */
+    private static int indexTask = 0;
+
+    /**
+     * Sorts the type of Task that is input by the user and prints out the respective types.
+     * Based on the keyword "todo", "deadline" or "event".
+     * Has helper functions addToDoTask(), addEventTask() and addDeadlineTask()
+     *
+     * @param userInput the string input by the user.
+     * @param splitUserInputs array of string that have been split into two.
+     */
+    public static void sortTypeOfTask(String userInput, String[] splitUserInputs) {
+        boolean isToDo = splitUserInputs[0].equals("todo");
+        boolean isDeadlines = splitUserInputs[0].equals("deadline");
+        boolean isEvent = splitUserInputs[0].equals("event");
```

Comment:
1. Avoid using magic numbers. Consider putting the numbers into a descriptive variable name to make the code easier to understand. For example, `0` can be put inito a variable `COMMAND_INDEX`. 

1. Consider abstracting the logic into methods (i.e. SLAP more) to make the code more understandable.